Wikipedia/Code/WMFAppViewController.m

```diff
@@ -901,7 +901,9 @@ - (void)finishResumingApp {
                                                    completion:^{
                                                    }];
     } else if (locationAuthorized != [defaults wmf_locationAuthorized]) {
```

Comment:
To preserve the behavior where location and continue reading aren't updated if there's a full update, could be split here as

```
} else {
  if (location) {
  }
  
  if (shouldShowOnResume) {
  }
}
```